### PR TITLE
clearQueue_newPort

### PR DESCRIPTION
**Pull request by Sidddddarth**
**Fixes** # (*issue*)

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
## Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added unit tests for the code
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by chandumlg — APPROVED** [review]

File: gateway/gateway.go [hunk 1]
```diff
@@ -1105,6 +1104,26 @@ func (gateway *HandleT) StartWebHandler() {
...
+	gateway.logger.Infof("Starting AdminHandler in %d", adminWebPort)
+	srvMux := mux.NewRouter()
+	srvMux.Use(headerMiddleware)
+	srvMux.HandleFunc("/v1/clear", gateway.stat(gateway.ClearHandler)).Methods("POST")
+	srvMux.HandleFunc("/v1/pending-events", gateway.stat(gateway.pendingEventsHandler)).Methods("POST")
+
+	srv := &http.Server{
+		Addr:              ":" + strconv.Itoa(adminWebPort),
+		Handler:           bugsnag.Handler(srvMux),
+		ReadTimeout:       config.GetDuration("ReadTimeOutInSec", 0*time.Second),
```
Comment (snarkychef) [inline gateway/gateway.go:1118]: We are using same configuration for timeouts in both servers? I see you have changed default value for WriteTimeout. but overriding the configuration in config.yaml will change both servers as they are using same config key right?

File: gateway/gateway.go [hunk 2]
```diff
@@ -1053,34 +1053,33 @@ func (gateway *HandleT) StartWebHandler() {
...
+		srvMux.HandleFunc("/schemas/event-versions", gateway.eventSchemaWebHandler(gateway.eventSchemaHandler.GetEventVersions)).Methods("GET")
+		srvMux.HandleFunc("/schemas/event-model/{EventID}/key-counts", gateway.eventSchemaWebHandler(gateway.eventSchemaHandler.GetKeyCounts)).Methods("GET")
+		srvMux.HandleFunc("/schemas/event-model/{EventID}/metadata", gateway.eventSchemaWebHandler(gateway.eventSchemaHandler.GetEventModelMetadata)).Methods("GET")
+		srvMux.HandleFunc("/schemas/event-version/{VersionID}/metadata", gateway.eventSchemaWebHandler(gateway.eventSchemaHandler.GetSchemaVersionMetadata)).Methods("GET")
+		srvMux.HandleFunc("/schemas/event-version/{VersionID}/missing-keys", gateway.eventSchemaWebHandler(gateway.eventSchemaHandler.GetSchemaVersionMissingKeys)).Methods("GET")
 	}
 
-	srvMux.HandleFunc("/v1/pending-events", gateway.stat(gateway.pendingEventsHandler))
-	srvMux.HandleFunc("/v1/clear", gateway.stat(gateway.ClearHandler))
+	srvMux.HandleFunc("/v1/pending-events", gateway.stat(gateway.pendingEventsHandler)).Methods("POST")
```
[the remark this answers is not in the document]
Comment (Sidddddarth) [reply gateway/gateway.go:1082]: to remain compatible with sources, keeping it. We'll remove it in next release

[the remark this answers is not in the document]
Comment (snarkychef) [reply gateway/gateway.go:1083]: 👍

File: gateway/gateway.go [hunk 3]
```diff
@@ -1053,34 +1053,34 @@ func (gateway *HandleT) StartWebHandler() {
...
+	srvMux.HandleFunc("/v1/alias", gateway.stat(gateway.webAliasHandler)).Methods("POST")
+	srvMux.HandleFunc("/v1/merge", gateway.stat(gateway.webMergeHandler)).Methods("POST")
+	srvMux.HandleFunc("/v1/group", gateway.stat(gateway.webGroupHandler)).Methods("POST")
+	srvMux.HandleFunc("/health", gateway.healthHandler).Methods("GET")
+	srvMux.HandleFunc("/v1/import", gateway.stat(gateway.webImportHandler)).Methods("POST")
+	srvMux.HandleFunc("/", gateway.healthHandler).Methods("GET")
+	srvMux.HandleFunc("/pixel/v1/track", gateway.stat(gateway.pixelTrackHandler)).Methods("GET")
+	srvMux.HandleFunc("/pixel/v1/page", gateway.stat(gateway.pixelPageHandler)).Methods("GET")
+	srvMux.HandleFunc("/version", gateway.versionHandler).Methods("GET")
+	srvMux.HandleFunc("/v1/webhook", gateway.stat(gateway.webhookHandler.RequestHandler)).Methods("POST")
```
Comment (arnab-p) [inline gateway/gateway.go:1070]: We should support POST and GET for the webhooks. Some of the destinations might send a GET request only

Comment (Sidddddarth) [reply gateway/gateway.go]: done

**Review by arnab-p — APPROVED** [review]

**Pull request merged**